packages/astro/test/postcss.test.js

```diff
@@ -0,0 +1,54 @@
+import { expect } from 'chai';
+import cheerio from 'cheerio';
+import eol from 'eol';
+import { loadFixture } from './test-utils.js';
+
+const PREFIXED_CSS = `-webkit-appearance:none;-moz-appearance:none;appearance:none`;
+
+let fixture;
+let bundledCSS;
+before(async () => {
+  fixture = await loadFixture({
+    projectRoot: './fixtures/postcss',
+    renderers: ['@astrojs/renderer-solid', '@astrojs/renderer-svelte', '@astrojs/renderer-vue'],
+  });
+  await fixture.build();
+
+  // get bundled CSS (will be hashed, hence DOM query)
+  const html = await fixture.readFile('/index.html');
+  const $ = cheerio.load(html);
+  const bundledCSSHREF = $('link[rel=stylesheet][href^=assets/]').attr('href');
+  bundledCSS = await fixture.readFile(bundledCSSHREF.replace(/^\/?/, '/'));
+});
+
+describe('PostCSS', () => {
+  it('works in Astro page styles', () => {
+    expect(bundledCSS).to.match(new RegExp(`.astro-page.astro-[^{]+{${PREFIXED_CSS}}`));
+  });
+
+  it('works in Astro component styles', () => {
+    expect(bundledCSS).to.match(new RegExp(`.astro-component.astro-[^{]+{${PREFIXED_CSS}}`));
+  });
+
+  it('works in <link>', () => {
+    expect(bundledCSS).to.match(new RegExp(`.a-n{${PREFIXED_CSS}}`));
+  });
+
+  it('works in JSX', () => {
+    expect(bundledCSS).to.match(new RegExp(`.solid{${PREFIXED_CSS}}`));
+  });
+
+  it('works in Vue', () => {
+    expect(bundledCSS).to.match(new RegExp(`.vue{${PREFIXED_CSS}}`));
+  });
+
+  it('works in Svelte', () => {
+    expect(bundledCSS).to.match(new RegExp(`.svelte.s[^{]+{${PREFIXED_CSS}}`));
+  });
+
+  it('ignores CSS in public/', async () => {
+    const publicCSS = await fixture.readFile('/global.css');
+    // neither minified nor prefixed
+    expect(eol.lf(publicCSS.trim())).to.equal(`.global {\n  appearance: none;\n}`);
```

Comment:
Bah Windows newlines strikes again! @matthewp what do you think about adding the `eol` package? 👍🏻 or 👎🏻 ? Happy to just do a `.replace()` here but figured we might need it in other places.

Reply:
this looks good to me!